Lib/Search/ElasticSearch/Client.php

```diff
@@ -162,9 +163,34 @@ public function removeDocuments(ClassMetadata $class, array $documents)
      */
     public function removeAll(ClassMetadata $class, $query = null)
     {
-        $type = $this->getIndex($class->getIndexForRead())->getType($class->type);
+        $index = $class->getIndexForRead();
+        if ($class->timeSeriesField) {
+            // CRUD APIs are single-index APIs. See https://www.elastic.co/guide/en/elasticsearch/reference/current/docs.html
```

Comment:
Correct. It's mentioned on the page linked to in the comment. The error I saw when testing it out was:
```
Elastica\Exception\Bulk\ResponseException: Error in one or more bulk request actions:
index: /test_revinate_search_bundle_time_series_*/status_logs/cJPNYaq-TkmSxMd-5v8Gvw 
caused InvalidIndexNameException[[test_revinate_search_bundle_time_series_*] 
Invalid index name [test_revinate_search_bundle_time_series_*], 
must not contain the following characters [\, /, *, ?, ", <, >, |,  , ,]]
```